README.md

```diff
@@ -1,13 +1,3 @@
-
-<p align="center">
-  <a href="https://travis-ci.com/github/username1565/text-encoding" target="_blank">
-    <img alt="Build status" src="https://travis-ci.com/username1565/text-encoding.svg?branch=master"  />
-  </a>
-  <a href="https://github.com/username1565/text-encoding/issues" target="_blank">
-    <img alt="Open Issues" src="https://badgen.net/github/open-issues/username1565/text-encoding" />
-  </a>
-</p>
-
```

Comment:
This images in the source code of this comment: https://github.com/username1565/text-encoding/pull/4#issue-416322338